includes/elements/container.php

```diff
@@ -294,9 +294,6 @@ protected function register_container_layout_controls() {
 			[
 				'type' => Controls_Manager::HIDDEN,
 				'default' => 'flex',
-				'selectors' => [
-					'{{WRAPPER}}' => '--display: {{VALUE}}',
-				],
 			]
```

Comment:
I have decided not to remove the hidden field, because we will need in future when we add the display; grid; to this widget. 

For moment we could however decide to remove this field completely. It doesn't make much of difference.

Reply:
You should remove the entire control. If we need it in the future we will use it.

Reply:
@udidol 

Removed